tools/propose_requests.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
```

Comment:
This file was moved to docker/k8s in this PR ?

Reply:
yes